vmpooler.yaml.example

```diff
@@ -11,20 +11,21 @@
 #     For multiple providers, specify one of the supported backing services (vsphere or dummy)
 #     (optional: will default to it's parent :key: name eg. 'vsphere')
 #
-#   - purge_unconfigured_folders
-#     Enable purging of VMs and folders detected within the base folder path that are not configured for the provider
-#     Only a single layer of folders and their child VMs are evaluated from detected base folder paths
-#     Nested child folders will not be destroyed. An optional whitelist can be provided to exclude folders
-#     A base folder path for 'vmpooler/redhat-7' would be 'vmpooler'
-#     Setting this on the provider will enable folder purging for the provider
+#   - purge_unconfigured_folders DEPRECATED, use purge_unconfigured_resources
+#   - purge_unconfigured_resources
+#     Enable purging of resources (typically VMs) or other items based on the provider. Provides a high-level cleanup
+#     mechanism for things that are live but not found in the vmpooler config ie in a pool config. See the provider's
+#     implementation for more details.
+#     Setting this on the provider will enable purging for the provider
 #     Expects a boolean value
 #     (optional; default: false)
 #
-#   - folder_whitelist
-#     Specify folders that are within the base folder path, not in the configuration, and should not be destroyed
-#     To exclude 'vmpooler/myfolder' from being destroyed when the base path is 'vmpooler' you would specify 'myfolder' in the whitelist
-#     This option is only evaluated when 'purge_unconfigured_folders' is enabled
-#     Expects an array of strings specifying the whitelisted folders by name
+#   - folder_whitelist DEPRECATED, use resources_allowlist
+#   - resources_allowlist
+#     Specify items names that should be ignored when purging. See the provider's
+#     implementation for more details.
+#     This option is only evaluated when 'purge_unconfigured_resources' is enabled
+#     Expects an array of strings specifying the allowed items by name
 #     (optional; default: nil)
```

Comment:
Since this will now vary by provider, should there even be a global one?

Reply:
there is no global one for resources_allowlist. Only the flag purge_unconfigured_resources is global (and overridable at the provider level)

Reply:
I must have misread this section - I thought it was at the global level.